scoper.inc.php

```diff
@@ -10,11 +10,15 @@
 
 use Isolated\Symfony\Component\Finder\Finder;
 
+$wp_classes   = json_decode( file_get_contents( 'vendor/sniccowp/php-scoper-wordpress-excludes/generated/exclude-wordpress-classes.json' ) );
+$wp_functions = json_decode( file_get_contents( 'vendor/sniccowp/php-scoper-wordpress-excludes/generated/exclude-wordpress-functions.json' ) );
+$wp_constants = json_decode( file_get_contents( 'vendor/sniccowp/php-scoper-wordpress-excludes/generated/exclude-wordpress-constants.json' ) );
```

Comment:
Should we check if these files exists before trying to use them? Maybe create a little helper function. 

```
function read_file( $path ){
      if( is_readable( $path ) ){
           return [];
      }
      return json_decode( file_get_contents( $path ) );
}
```

Reply:
There was a small discussion related to a helper function here --- but not related to the existence of the files 
https://github.com/GoogleForCreators/web-stories-wp/pull/10511#discussion_r802115710

Reply:
We're including a specific version of a package that is known to include these files.

If a file does not exist we absolutely need to be informed. If this fails we need to know.

Reply:
> If a file does not exist we absolutely need to be informed. If this fails we need to know.

1) To confirm we do want to add a helper?
2) If yes so we want to echo to the console or throw -- (how do we want to be informed?)

Reply:
No helper needed.

I don't see a need for action here.